client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/ExecuteOperation.java

```diff
@@ -68,4 +70,20 @@ protected void executeOperation(Channel channel) {
    public void acceptResponse(ByteBuf buf, short status, HeaderDecoder decoder) {
       complete(bytes2obj(channelFactory.getMarshaller(), ByteBufUtil.readArray(buf), dataFormat.isObjectStorage(), cfg.getClassAllowList()));
    }
+
+   @Override
+   protected void addParams(StringBuilder sb) {
```

Comment:
This seems a bit unrelated, but fine

Reply:
Without this change, cache creation operations are logged as `AdminOperation (default)`, with no mention of the cache name.